client/components/Nav.js

```diff
@@ -62,13 +62,10 @@ class Nav extends Component {
             <NavLink to="/admin/products" className="nav-item" activeClassName="active">
               Products
             </NavLink>
-            <NavLink to="/admin/categories" className="nav-item">
+            <NavLink to="/admin/categories" className="nav-item" activeClassName="active">
```

Comment:
I believe I set this up on the Nav for other links.  I have had problems with getting this to work in the past and not sure why.  Is it working in this instance?